src/index.html

```diff
@@ -27,13 +25,10 @@
     crossorigin="anonymous">
   </script>
   <script 
-    src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" 
-    integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" 
-    crossorigin="anonymous">
+    src="//cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js">
   </script>
```

Comment:
May I know why you deleted the CDN to get Bootstrap v4?. We need it in order for Bootstrap v4 to work properly